src/blockchain/TxStatus.ts

```diff
@@ -5,11 +5,17 @@
 /**
  * Dummy comment needed for correct doc display, do not remove
  */
+
+import { Codec } from '@polkadot/types/types'
+import ICType from '../types/CType'
+
+export type QueryResult = Codec | undefined | null | ICType | string
+
 export default class TxStatus {
   public type: string
-  public payload: any
+  public payload: QueryResult
 
-  public constructor(type: string, payload?: any) {
+  public constructor(type: string, payload?: QueryResult) {
```

Comment:
TxStatus.payload is only ever used by Ctype.store - unfortunately, it's used as a plain object, so QueryResult won't do. Would be worthwhile discussing whether payload is needed at all